src/debug/di/module.cpp

```diff
@@ -880,6 +881,11 @@ HRESULT CordbModule::InitPublicMetaDataFromFile(const WCHAR * pszFullPathName,
                                                 DWORD dwOpenFlags,
                                                 bool validateFileInfo)
 {
+#ifdef FEATURE_PAL    
+    // TODO: Some intricate details of file mapping don't work on Linux as on Windows.
+    // We have to revisit this and try to fix it for POSIX system. 
+    return E_FAIL;
+#else    
```

Comment:
As I recall, this entire scheme is a performance optimization to save virtual memory. We might want to revisit the scheme as a whole and decide if we should do something more abstract, or fix it, or not support the option. I'm fine disabling it as the first step : )